python/ray/autoscaler/_private/commands.py

```diff
@@ -178,6 +180,17 @@ def request_resources(
         to_request += [{"CPU": 1}] * num_cpus
     if bundles:
         to_request += bundles
+
```

Comment:
I feel it's hard to understand here. I thought we move things to RPC so that we won't depend on KV in autoscaler side. Why do we put it into the kv?

Reply:
Why don't you just have a gRPC endpoint in GCS monitor and store the info there? In GCS monitor, you can write the data into KV to persist it.

Reply:
fair point